fpl/data/data_converter_class.py

```diff
@@ -0,0 +1,80 @@
+"""Convert data to CSV."""
+import json
+from pathlib import Path
+from tqdm import tqdm
+import pandas as pd
+
+
+class DataConverter:
+    """Data converter class."""
+
+    def __init__(
+        self, data_dir: str, entity: str, main_data_folder="data", main_interim_folder="interim"
+    ):
+        """Set configuration parameters.
+
+        Args:
+            data_dir (str): Path to dir holding JSON dumps of Fantasy Premier League.
+            entity (str): Either "elements" or "teams".
+        """
+        self.data_dir = Path(data_dir)
```

Comment:
Should we set `data_dir` as optional with default `None` and if set to `None` then we set `data_dir` to interim path (similar to what is done in `Blob_Importer`?

Reply:
Agree, I overlooked your util-file